tools/main.toit

```diff
@@ -193,6 +231,16 @@ apply_file_patch_ --patch_path/string --file_path/string:
   writer.write patch
   writer.close
 
+update_patch_ --from/string --to/string --output/string:
+  print "updating $output"
```

Comment:
Is the debug printing here consistent with other printing in this code (capitalization, period at end)?

Reply:
Using `ui.print` now. Capitalized. With ".".
done.